tests/test-util/test.manifest.js

```diff
@@ -35,6 +38,15 @@ describe('util/manifest', () => {
         })
     ));
 
+    it('allows manifests without IDs', () => withTempDir(
```

Comment:
@kumar this is similar to the test cases that starts from line 136, how about move it near there? 

by reading the test description, it seems the same test from line 136, but it is actually different, in fact it tests that a manifest without the entire "applications" section is still valid, and so it could be changed to have a more explicit description (e.g. "allows manifests without the 'applications' property", and maybe renaming `manifestWithoutId` manifestWithoutApplications` as well?)